downstream/assemblies/platform/assembly-custom-inventory-scripts.adoc

```diff
@@ -0,0 +1,25 @@
+[id="assembly-custom-inventory-scripts"]
+
+= Custom Inventory Scripts
+
+[NOTE]
+====
+Inventory scripts have been discontinued. 
+
+For more information, see [Export old inventory scripts] in the _Automation Controller User Guide.
+====
+
+If you use custom inventory scripts, migrate to sourcing these scripts from a project, refer to [Inventory File Importing], and to [Inventory Sources] in the _Automation Controller User Guide_.
```

Comment:
Suggested re-write:

If you use custom inventory scripts, migrate to sourcing these scripts from a project. For more information, see Inventory File Importing], and to [Inventory Sources] in the _Automation Controller User Guide_.